modules/client/examples/06-multisig-client/06-get-members.ts

```diff
@@ -0,0 +1,26 @@
+/* MARKDOWN
+### Loading the list of members (multisig plugin)
+*/
+import { MultisigClient, Context, ContextPlugin } from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const daoAddressorEns = "0x12345...";
+
+const memebers: string[] = await client.methods.getMembers(daoAddressorEns);
+console.log(memebers);
+/*
```

Comment:
Do we have a way to know how many approvals are needed? This might look more like `getPluginSettings` than just getting the addresses

Reply:
Yes we can change this with the new parameters, I assumed only addresses will be needed, renaming this to `getPluginSettings` and returning something like this would make more sense:

```
type MultisigPluginSettings {
  addressesOrEns: string[]
  votingSettings: MultisigVotingSettings
}

type MultisigVotingSettings {
  executeOnPass?: boolean // default true
  minSupport: number // 0-1
}
```

Let me know what do you think